saleor/graphql/schema.graphql

```diff
@@ -380,6 +380,41 @@ type AppUpdate {
   app: App
 }
 
+enum AreaUnitsEnum {
+  SQ_CHAIN
+  SQ_CHAIN_BENOIT
+  SQ_CHAIN_SEARS
+  SQ_BRITISH_CHAIN_BENOIT
+  SQ_BRITISH_CHAIN_SEARS
+  SQ_BRITISH_CHAIN_SEARS_TRUNCATED
+  SQ_CM
+  SQ_BRITISH_FT
+  SQ_BRITISH_YD
+  SQ_CLARKE_FT
+  SQ_CLARKE_LINK
```

Comment:
Clarke's units are special cases of the generic imperial units. Links are obsolete, they are a subunit of chains.